README.md

```diff
@@ -70,21 +70,21 @@ If you use Terraform to manage your infrastructure, MongoDB for VS Code helps yo
-| `mdb.useDefaultTemplateForPlayground`                                    	| Choose whether to use the default template for playground files or to start with an empty playground editor.                                                                                                                         	| `true`                                                                                                                     	|
-| `mdb.uniqueObjectIdPerCursor`                                            	| The default behavior is to generate a single ObjectId and insert it on all cursors. Set to true to generate a unique ObjectId per cursor instead.                                                                                    	| `false`                                                                                                                    	|
-| `mdb.sendTelemetry`                                                      	| Opt-in and opt-out for diagnostic and telemetry collection.                                                                                                                                                                          	| `true`                                                                                                                     	|
+| `mdb.shell` | The MongoDB shell to use ( `mongosh`  or the legacy  `mongo` ).              | `mongosh` |
+| `mdb.showMongoDBConnectionExplorer` | Show or hide the MongoDB Connection explorer.    | `true`  |
+| `mdb.showMongoDBPlaygrounds` | Show or hide the MongoDB Playgrounds explorer. | `true` |
+| `mdb.showMongoDBHelpExplorer` | Show or hide the MongoDB Help panel. | `true` |
+| `mdb.defaultLimit` | The number of documents to fetch when viewing documents from a collection. | `10` |
+| `mdb.confirmRunAll` | Show a confirmation message before running commands in a playground. | `true` |
+| `mdb.confirmDeleteDocument` | Show a confirmation message before deleting a document in the tree view. | `true` |
+| `mdb.excludeFromPlaygroundsSearch` | Exclude files and folders while searching for playground files in the current workspace. | Refer to [`package.json`](https://github.com/mongodb-js/vscode/blob/7b10092db4c8c10c4aa9c45b443c8ed3d5f37d5c/package.json) |
+| `mdb.connectionSaving.hideOptionToChooseWhereToSaveNewConnections` | When a connection is added, a prompt is shown that let's the user decide where the new connection should be saved. When this setting is checked, the prompt is not shown and the default connection saving location setting is used. | `true` |
```

Comment:
```suggestion
| `mdb.connectionSaving.`  `hideOptionToChooseWhereToSaveNewConnections` | When a connection is added, a prompt is shown that let's the user decide where the new connection should be saved. When this setting is checked, the prompt is not shown and the default connection saving location setting is used. | `true` |
```

Reply:
Two spaces are a replacement for `<br>` to break down a long text and make table rows look equal in width.